index.bs

```diff
@@ -2369,6 +2369,24 @@ To <dfn>render viewport to a canvas</dfn> given |viewport| and |rect|:
 
 </div>
 
+<div algorithm>
+To <dfn>await the next animation frame</dfn> given |window|:
+
+1. Let |render id| be the string representation of a
```

Comment:
Let's do `opaque implementation-specific string`. That way implementations aren't bound by UUID.

Reply:
I note in passing that render id isn't observable either way, so making it a UUID in the spec or not doesn't actually constrain implementations.

Reply:
As long as it doesn't matter either way, I'm happy to take @jrandolf's suggestion since that seems less distracting (though I used HTML's wording: "new unique opaque string").